docs/recipes/Code-Splitting-Using-Holocron.md

```diff
@@ -1,7 +1,276 @@
+  * [Using `holocronModule`](#using-holocronModule)
+  * [Using `Suspense` and `lazy`](#using-suspense-and-lazy)
+  * [How It Works](#how-it-works)
+  * [Performance Considerations](#performance-considerations)
+  * [Include](#include-existing-tools-into-the-pipeline)
+
+For One App, Holocron modules are a formal way of code-splitting for a modern,
+micro front-end in the One App ecosystem. Utilizing the module map as a system
+which promotes synergy and asynchrony across teams and collaborators on
+a single application.
+
+With Holocron each individual module is by definition, a code split bundled
+component with language packs that can be combined and composed based on our
+needs. Each module is versioned, configurable, routable and can be further
+split into smaller chunks using dynamic importing.
+
+The main advantage we get with code splitting our modules is controlling what parts of
+a module is first delivered and larger chunks are loaded by user actions and are not
+necessary to include at initial delivery. This can save end users kilobytes of data
+transfer (until needed) and reduce the time for waiting on the Holocron module
+bundle to load.
+
+Like `@loadable/component` and `react-loadable`, `holocron` is a complete code-splitting
+solution that works in both client and server side, however there is no need to scaffold
+a build infrastructure to support code-splitting when we use `one-app-bundler`.
+
+**📘 More Information**
+
+* **[`holocron`][holocron-readme]**
+* **[`one-app-bundler`][one-app-bundler-readme]**
+
+## Route Based Code Splitting
+
+A Holocron module can be route-driven using `ModuleRoute` and we can use this component
+to shape our routing for One App. Since each Holocron module is an independent bundle,
+our `childRoutes` is a map of code-split modules that is driven by the url.
+
+`ModuleRoute` is an extension of `Route` from [`one-app-router`][one-app-router-readme]
+and can render the module(s) by its route per `history`.
+Our Holocron modules are now loaded based on routing through the application.
+
+```jsx
+import React from 'react';
+import { RenderModule } from 'holocron';
+import { ModuleRoute } from 'holocron-module-route';
+import { IndexRedirect } from '@americanexpress/one-app-router';
+
+export const childRoutes = () => (
+  <ModuleRoute moduleName="diner-finder">
+    <IndexRedirect to="map" />
+    <ModuleRoute path="map" moduleName="diner-finder-map" />
+    <ModuleRoute path="selection" moduleName="diner-finder-selection-container">
+      <ModuleRoute moduleName="diner-finder-selection" />
+    </ModuleRoute>
+  </ModuleRoute>
+);
+```
+
+> To view a root Holocron module with a childConfig, [check out the sample module "Frank Lloyd Root"][frank-lloyd-root]
+
+**📘 More Information**
+
+* **[`holocron-module-route`][holocron-module-route-readme]**
+* **[`one-app-router`][one-app-router-readme]**
+
+## Holocron Module Chunks _Via_ `import()`
```

Comment:
It needs to be clearer here and in the prod-sample that this should _only_ be used client side, e.g. triggered by user action

Reply:
Will update

Reply:
updated in 32a3038